### PR TITLE
AO3-4420 Correct pluralisation on kudos emails

### PR DESCRIPTION
**Pull request by zz9pzza**
https://otwarchive.atlassian.net/browse/AO3-4420

Test by having a user receive a kudos from one person and then from two

### CODE REVIEW

File: config/locales/en-US.yml [hunk 1]
```diff
@@ -242,8 +242,9 @@
           html: Sent by <b>%{sent_by}</b>.
           text: Sent by %{sent_by}.
     kudos:
-      guest: A guest
-      guests: guests
+      guest: 
+        one: a guest
```
Comment (sarken) [inline config/locales/en-US.yml]: I'm having a hard time figuring it out -- is this for the sentence "A guest left kudos on %{work}" or "%{usernames} and a guest left kudos on %{work}"? If it's for the first one, it should be "A guest," not "a guest."

Comment (zz9pzza) [inline config/locales/en-US.yml]: This gives me a headache too:

https://github.com/otwcode/otwarchive/blob/master/app/mailers/kudo_mailer.rb#L35

        if !names.nil? && names.size > 0
          kudo_givers = names
          kudo_givers << guest_kudos(guest_count) unless guest_count == 0
        else
          kudo_givers << guest_kudos(guest_count).capitalize unless guest_count == 0
        end
        next if kudo_givers.empty?

See we have the capitalize so a guest  is correct as the A or the first letter of the list of users is capitalized.

Comment (sarken) [inline config/locales/en-US.yml]: Okay, great!

**Comment by sarken** [comment]
Whoops, taking off Ready to Merge label because @priscilladc has some comments still!

**Comment by priscilladc** [comment]
Specific comments: we'd need 'kudos' to be variable (so like, someone left [gold star], two someones left [gold stars]); we also need 'left' to be variable (imagine like, he [leaves] kudos, they [leave] kudos; the verb changes when it's a different number of people).

More generally, and this is an underlying problem: we should do our best to avoid strings that are built like $variable + $variable + $variable, because languages structure sentences differently. Not all languages will order this sentence like someone + left + kudos. Latin, for example, would do someone + kudos + left. German, to be even more entertaining, apparently goes '[person] hat Kudos hinterlassen' — the 'kudos' goes smack dab in the *middle* of the verb. 

tl;dr we need to be able to translate the different strings in, sorry, not just the bits!

**Comment by priscilladc** [comment]
Ah, sorry! Just noticed #2282. Take back the comment about kudos then, since it's waiting to be merged :D but the rest still applies.

**Comment by zz9pzza** [comment]
@priscilladc @sarken I don't think this needs any more work done on it ?

**Comment by zz9pzza** [comment]
@sarken can we put ready to merge back on this ?

**Comment by sarken** [comment]
Done!

**Comment by sarken** [comment]
Just a heads up that this has gone grey and will need to be updated.
